### PR TITLE
Minor fixes for finding CUDNN

### LINKED ISSUE #8485: Build Error lib.obj not found
Dear community,

i encountered a build error i can not resolve.  I'm trying to build pytorch with the specs below. The error i'm encountering is this one:

img

## Issue description

Obviously it's lacking a dubious lib.obj for linking. The question is why it is searching for it in the standard Nvidea folder. Shouldn't the files compiled not be safed somewhere else. Furthermore there is no other error before this one in the build process. 

Can anyone help me with what has gone wrong here. I followed the instructions of the regular link:
[pytorch from source](https://github.com/pytorch/pytorch#from-source)

Best regards

## System Info

- PyTorch 
- How you installed PyTorch (conda, pip, source): Source
- Build command you used (if compiling from source): python setup.py install
- OS: Windows 10
- PyTorch version: 0.4
- Python version: 3.6
- CUDA/cuDNN version: 9.0
- GPU models and configuration: GTX 1070
- Visual Studio 17 (15.7.3)
- CMake version: 3.11.1

### PR DESCRIPTION
**Pull request by peterjc123**
Fixes #8485 probably.

### CODE REVIEW

File: tools/setup_helpers/cudnn.py [hunk 1]
```diff
@@ -34,6 +32,9 @@
         'C_INCLUDE_PATH',
         'CPLUS_INCLUDE_PATH',
     ])))
+    # Add CUDA related dirs to candidate list
+    for suffix in LIB_SUFFIXS:
+        lib_paths.append(os.path.join(CUDA_HOME, suffix))
```
Comment (soumith) [inline tools/setup_helpers/cudnn.py]: the ordering in lib_paths matters. You've appended `CUDA_HOME + suffix` entries to `lib_paths`, which means they will be considered after system paths like `/usr/lib/x86_64-linux-gnu/`. What you need to do is insert them right after os.getenv('CUDNN_LIB_DIR')

File: tools/setup_helpers/cudnn.py [hunk 2]
```diff
@@ -10,13 +10,11 @@
 CUDNN_INCLUDE_DIR = None
 CUDNN_LIBRARY = None
 WITH_STATIC_CUDNN = os.getenv("USE_STATIC_CUDNN")
+LIB_SUFFIXS = ['lib', 'lib64', 'lib/x64']
```
Comment (soumith) [inline tools/setup_helpers/cudnn.py]: it's `SUFFIXES`, not `SUFFIXS`

**Review by soumith — CHANGES_REQUESTED** [review]
see inline comments

File: tools/setup_helpers/cudnn.py (hunk 2 above)
Comment (peterjc123) [reply tools/setup_helpers/cudnn.py]: Yes, will fix!

**Review by soumith — APPROVED** [review]

**Comment by soumith** [comment]
lint needs to be fixed

**Comment by peterjc123** [comment]
The CI failure seems to have nothing to do with this PR.

**Comment by soumith** [comment]
let's wait for the pytorch CI to pass

**Comment by peterjc123** [comment]
@soumith The pytorch CI tests all passed.

**Review by soumith — APPROVED** [review]

**Pull request merged**